apps/omg_watcher/test/omg_watcher/exit_processor/core/standard_exit_challenge_test.exs

```diff
@@ -194,6 +193,23 @@ defmodule OMG.Watcher.ExitProcessor.Core.StandardExitChallengeTest do
                |> Core.create_challenge(processor)
     end
 
+    test "creates challenge: tx utxo double spent not in IFE, but there is an unrelated IFE open",
```

Comment:
Is this a negative test? Unrelated IFE being open does not affect normal SE double-spend challenge?

Reply:
> tx utxo double spent not in IFE

is confusing a bit